api/src/main/java/org/openmrs/module/esaudereports/reporting/library/cohort/QualityImprovementCohort.java

```diff
@@ -33,13 +36,18 @@
 	 * @return CohortDefinition
 	 */
 	public CohortDefinition clinicalConsulationsNumerator() {
-		SqlCohortDefinition cd = new SqlCohortDefinition();
-		String sqlQuery = "";
-		cd.setName("Clinical consultation numerator");
-		cd.addParameter(new Parameter("startDate", "Start date", Date.class));
-		cd.addParameter(new Parameter("endDate", "End date", Date.class));
-		cd.addParameter(new Parameter("location", "Location", Location.class));
-		cd.setQuery(sqlQuery);
+		CompositionCohortDefinition cd = new CompositionCohortDefinition();
+		cd.setName("INSCRITOS NO PERIODO DE INCLUSAO E QUE TIVERAM CONSULTA CLINICA NO MESMO DIA DA INSCRIÇÃO");
+		cd.setDescription("São pacientes inscritos no periodo de inclusao e que tiveram consulta clinica no mesmo dia da inscrição");
+		cd.addParameter(new Parameter("startDate", "Data Inicial", Date.class));
+		cd.addParameter(new Parameter("endDate", "Data Final", Date.class));
+		cd.addParameter(new Parameter("revisionEndDate", "Data Final de Revisão", Date.class));
+		cd.addParameter(new Parameter("location", "US", String.class));
```

Comment:
For this line you need this **cd.addParameter(new Parameter("location", "Location", Location.class));** so that it can load locations from openmrs. The location paramaetr is linked to encounters